src/Avalonia.Visuals/Platform/IPlatformRenderInterface.cs

```diff
@@ -84,6 +85,10 @@ public interface IPlatformRenderInterface
         /// <returns>An <see cref="IWriteableBitmapImpl"/>.</returns>
         IWriteableBitmapImpl CreateWriteableBitmap(PixelSize size, Vector dpi, PixelFormat? format = null);
 
+        IBitmapImpl LoadBitmap(string fileName, BitmapDecodeOptions decodeOptions);
```

Comment:
Maybe rather than having two extra overloads, add the options parameter to the existing overloads and pass null for default options?

Reply:
done